bin/fixmystreet.com/one-off-status-update

```diff
@@ -78,13 +78,26 @@ if (!$commit) {
     say "*** DRY RUN ***";
 }
 
-my %alloy_to_fms_map = (
+my %enq_to_fms_map = (
+    'Issued to Inspector' => 'investigating',
+    'In Progress' => 'investigating',
+    'Completed' => 'closed',
+);
+
+my %rfc_to_fms_map = (
     'No Action Necessary' => 'unable to fix',
     'Outside NCC Control' => 'not responsible',
     'Highways to Monitor. No Action' => 'unable to fix',
     'Work Instructed' => 'in progess',
 );
 
+my %defect_to_fms_map = (
+    'Remedied' => 'fixed - council',
+    'Order Raised' => 'in progress',
+    'Rejected' => 'closed',
+    'Found and Forwarded' => 'closed',
```

Comment:
Should these two be in the live config also then? As that defaults to open on these two.